app/src/main/java/org/redcross/openmapkit/OSMMapBuilder.java

```diff
@@ -34,8 +34,9 @@
  * nhallahan@spatialdev.com* 
  */
 public class OSMMapBuilder extends AsyncTask<File, Long, JTSModel> {
-    
-    private static final float MIN_VECTOR_RENDER_ZOOM = 18;
+
+    // Render vectors at lower zoom - useful for showing distribution.
+    private static final float MIN_VECTOR_RENDER_ZOOM = 12;
```

Comment:
We don't want to change this constant, because with typical data sets that we have been using in the app, the renderer has trouble at lower zooms, and the app may crash.

Perhaps allow this to be set in a settings file?